e2e/cypress/integration/onboarding/cloud_sysadmin_onboarding_spec.js

```diff
@@ -7,15 +7,13 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Group: @enterprise @onboarding @cloud_only
+// Group: @onboarding
 
-describe('Cloud Onboarding - Sysadmin', () => {
+describe('Onboarding - Sysadmin', () => {
```

Comment:
The happy path test failed on me. Could you please take a look? Thanks!
<img width="572" alt="Screen Shot 2021-08-16 at 2 26 34 PM" src="https://user-images.githubusercontent.com/5334504/129520596-fa3f7d79-7b22-4ea3-b387-50a6045f817b.png">